### DISCUSSION TITLE
.trivyignore support

### DISCUSSION THREAD

**Discussion by thrivikramgit** [aquasecurity/trivy#4934] [Q&A] [answered]
### Question

Hello Team,
Currently I am scanning the list of Images using trivy and would like to provide different .trivyignore for each image. I went through the document but didn't found any resources. Could you please help me?

Thanks for your support

### Target

Container Image

### Scanner

Vulnerability

### Output Format

JSON

### Mode

Standalone

### Version

```bash
Version: 0.44.0
```

**Comment by knqyf263** [comment, marked as answer]
You can pass different paths.

>      --ignorefile string                 specify .trivyignore file (default ".trivyignore")

https://aquasecurity.github.io/trivy/v0.44/docs/references/configuration/cli/trivy_image/
